salt/utils/parsers.py

```diff
@@ -654,7 +661,9 @@ def __setup_logfile_logger_config(self, *args):  # pylint: disable=unused-argume
             self.config.pop(self._logfile_loglevel_config_setting_name_)
```

Comment:
Oh! I see what you mean. Those were good pointers. I figured out how to get this resolved. Thanks!